client/my-sites/checkout/get-thank-you-page-url/test/get-thank-you-page-url.ts

```diff
@@ -1013,7 +1013,10 @@ describe( 'getThankYouPageUrl', () => {
 		expect( url ).toBe( `/checkout/foo.bar/offer-plan-upgrade/business/${ samplePurchaseId }` );
 	} );
 
-	it( 'redirects to business monthly upgrade nudge if jetpack is not in the cart, and premium monthly is in the cart', () => {
+	// NOTE: as part of the calypso_postpurchase_upsell_monthly_to_annual_plan experiment
+	// we're disabling the monthly premium to monthly business plan upsell
+	it( 'redirects to thank you page if jetpack is not in the cart, and premium monthly is in the cart', () => {
+		// it( 'redirects to business monthly upgrade nudge if jetpack is not in the cart, and premium monthly is in the cart', () => {
```

Comment:
Unsure what's the best way to proceed with this, as we are changing the behavior for the experiment

Reply:
I kind of feel like we could work as if the Premium-Business monthly upsell removal is permanent (if that makes things easier to work with). Naturally, we can always change our mind after the experiment, and put it back, referring to these old PRs if needed.